src/autoupdater.js

```diff
@@ -195,6 +195,16 @@ class AutoUpdater {
       }
     }
 
+    const exclidedLabels = this.config.excludedLabels();
+    for (const label of pull.labels) {
+      if (exclidedLabels.includes(label.name)) {
+        ghCore.info(
+          `Pull request has excluded label '${label.name}', skipping update.`
+        );
+        return false;
```

Comment:
There's a chance this may not work if `PR_FILTER=labelled` and a label matches - see line 167 above.

Would it make more sense if this exclusion check was done before anything else? i.e. moving this block up to just below line 143, before the `if (prFilter === 'labelled')` check.